webapp/security/views.py

```diff
@@ -333,5 +342,208 @@ def cve(cve_id):
+    @params json: the body of the request
+    """
+
+    data = flask.request.get_json()
+    response = flask.jsonify({"message": "Unable to get body"}), 400
+    packages = []
+    references = []
+    bugs = []
+
+    # Check if CVE exists by candidate
+    if db_session.query(CVE).filter(CVE.id == data["id"]).count() > 0:
+        response = flask.jsonify({"message": "CVE already exists"}), 400
+        return response
+
+    # Packages
+    # Check if there are packages before mapping
+    if "packages" in data and len(data["packages"]) > 0:
+        for pkg in data["packages"]:
+            releases = []
+            for rel in pkg["releases"]:
+                release = PackageReleaseStatus(
+                    name=rel["name"],
+                    status=rel["status"],
+                    status_description=rel["status_description"],
+                )
+            releases.append(release)
+            package = Package(
+                name=pkg["name"],
+                source=pkg["source"],
+                ubuntu=pkg["ubuntu"],
+                debian=pkg["debian"],
+                releases=releases,
+            )
+            packages.append(package)
+
+    if "references" in data and len(data["references"]) > 0:
+        for ref in data["references"]:
+            reference = CVEReference(uri=ref)
+            references.append(reference)
+
+    if "bugs" in data and len(data["bugs"]) > 0:
+        for b in data["bugs"]:
+            bug = Bug(uri=b)
+            bugs.append(bug)
+
+    cves = [
+        CVE(
+            id=data["id"],
+            status=data["status"] if "status" in data else "",
+            last_updated_date=data["last_updated_date"]
+            if "last_updated_date" in data
+            else None,
+            public_date_usn=data["public_date_usn"]
+            if "public_date_usn" in data
+            else None,
+            priority=data["priority"] if "priority" in data else None,
+            cvss=data["cvss"] if "cvss" in data else "",
+            assigned_to=data["assigned_to"] if "assigned_to" in data else "",
+            discovered_by=data["discovered_by"]
+            if "discovered_by" in data
+            else "",
+            approved_by=data["approved_by"] if "approved_by" in data else "",
+            description=data["description"] if "description" in data else "",
+            ubuntu_description=data["ubuntu_description"]
+            if "ubuntu_description" in data
+            else "",
+            notes=data["notes"] if "notes" in data else "[]",
+            packages=packages,
+            references=references,
+            bugs=bugs,
+        )
+    ]
+
+    try:
+        db_session.add_all(cves)
+        db_session.commit()
+    except exc.SQLAlchemyError as e:
+        response = flask.jsonify({"message": e}), 400
+        return response
+
+    response = flask.jsonify({"message": "CVE created succesfully"}), 200
+    return response
```

Comment:
Again, you don't need the extra variable:

```suggestion
    return  flask.jsonify({"message": "CVE created succesfully"}), 201
```